drivers/azure/create.go

```diff
@@ -3,3 +3,116 @@
 // that can be found in the LICENSE file.
 
 package azure
+
+import (
+	"context"
+	"github.com/drone/autoscaler"
+	"github.com/rs/zerolog/log"
+	"fmt"
+	"time"
```

Comment:
best practice for import ordering  https://github.com/golang/go/wiki/CodeReviewComments#imports